src/senselab/utils/tasks/cross_correlation.py

```diff
@@ -31,19 +31,19 @@ def compute_normalized_cross_correlation(signal1: torch.Tensor, signal2: torch.T
         raise ValueError("Input signals must be one-dimensional")
 
     # Convert PyTorch tensors to NumPy arrays
-    signal1 = signal1.numpy()
-    signal2 = signal2.numpy()
+    signal1_np: np.ndarray = signal1.numpy()
```

Comment:
again, is this supposed to be in this PR or is it just a weird Github thing happening?

Reply:
same as above. sorry